### DISCUSSION TITLE
Add additional fonts

### DISCUSSION THREAD

**Discussion by Airradda** [Divested-Mobile/DivestOS-Build#122] [Q&A] [answered]
I'm trying to add the [OpenDyslexic](https://opendyslexic.org/) font. Is there a way to add additional fonts to the system? I've looked around but all the methods found don't work for DivestOS or require root to do so.

**Comment by SkewedZeppelin** [comment, marked as answer]
Fonts have to be compiled in

You can see how LineageOS added some fonts here:
https://github.com/LineageOS/android_packages_overlays_Lineage/tree/lineage-19.1/fonts
https://android.googlesource.com/platform/external/google-fonts
https://android.googlesource.com/platform/external/google-fonts/lato/+/refs/heads/master

However do note fonts only apply to the system components and only a few apps.
Most apps will override the font.
